size-auditor/size-auditor.schema.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Size auditor Configuration",
+  "description": "Configuration file for the Size Auditor tool",
+  "type": "object",
+  "properties": {
+    "sizeTolerance": {
+      "description": "The tolerance limit above which Size Auditor would fail a pull request",
+      "type": "number"
+    },
+    "trackingEntity": {
+      "description": "The entity that is audited for size regressions by Size Auditor",
```

Comment:
Perhaps clarify the differences per enum value or link to documentation that does. I'm not sure if there's a pattern for this. I'm not certain as someone unfamiliar with the area.

Also is "Assembly" an external concept?